<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MatrixCal</class>
 <widget class="QDialog" name="MatrixCal">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>715</width>
    <height>301</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>矩阵运算</string>
  </property>
  <property name="windowIcon">
   <iconset resource="painter/cal.qrc">
    <normaloff>:/images/cal.png</normaloff>:/images/cal.png</iconset>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_8">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label_index">
           <property name="text">
            <string>选择计算方式:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="comboBox">
           <item>
            <property name="text">
             <string>矩阵相加</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>矩阵相减</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>矩阵相乘</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>矩阵转置</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>数乘矩阵</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label_rowCount">
           <property name="text">
            <string>行数:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="spinBox_rowCount"/>
         </item>
         <item>
          <widget class="QLabel" name="label_colCount">
           <property name="text">
            <string>列数:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="spinBox_colCount"/>
         </item>
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>系数:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDoubleSpinBox" name="doubleSpinBox"/>
         </item>
         <item>
          <widget class="QPushButton" name="btn_ok">
           <property name="text">
            <string>确定</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Minimum</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_7">
       <item>
        <widget class="QGroupBox" name="groupBox_A">
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>200</height>
          </size>
         </property>
         <property name="title">
          <string>矩阵A</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QTableWidget" name="tableWidget_A">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_B">
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>200</height>
          </size>
         </property>
         <property name="title">
          <string>矩阵B</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QTableWidget" name="tableWidget_B">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_equal">
         <property name="text">
          <string>=</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_C">
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>200</height>
          </size>
         </property>
         <property name="title">
          <string>矩阵C</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <item>
           <widget class="QTableWidget" name="tableWidget_C">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="painter/cal.qrc"/>
 </resources>
 <connections/>
</ui>
